assets/styles/components/_pagination.scss

```diff
@@ -31,7 +31,7 @@
   .prev {
     float: left;
 
-    @include respond-to('medium') {
+    @include respond-to('large') {
```

Comment:
Generally speaking, I don't really like this solution. Just because @luukdv's theoretical proposal causes a practical issue (specifity issue in this case), I don't think this should be solved this way. I'd recommend dealing with the specifity issue.

In this case, showing pagination on screens smaller than `981px` could come in handy for users.